app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,11 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find_by_id(params[:user_id])
+    @posts = @user.posts
+  end
```

Comment:
Good job having all the setups with your controllers, however when I try to create a post, it doesn't work and this makes it impossible to know if your controller actions are working, consider adding this `  t.integer :author_id` to your post migration file in order for posts to be created successfully.


[screen-capture (1).webm](https://github.com/gaks1/ruby-blog-app/assets/84930355/1767c04f-faf4-40c4-8757-454f9ec7c168)

Reply:
I believe it is because of the post_counter it seems it needs to be initialized as zero

Reply:
@DuaneDave

Reply:
the comments_counter and likes_counter also need to be zero like    first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my 1 post', comments_counter: 0, likes_counter: 0)

Reply:
@DuaneDave

Reply:
the comments_counter and likes_counter also need to be zero like first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my 1 post', comments_counter: 0, likes_counter: 0) @DuaneDave